community/assessments/projects/openebs/self-assessment.md

```diff
@@ -34,13 +34,15 @@ Author: Niladri Halder ([@niladrih](https://github.com/niladrih))
 
 ## Metadata
 
+| | |
```

Comment:
This renders funny? https://github.com/cncf/tag-security/blob/d3437f9050595ba65b45e24e39586ba659613fb0/community/assessments/projects/openebs/self-assessment.md#metadata

Reply:
It does do that. The template comes with this style, and it looks a bit better on the website. Going with this for now.